Modules/Scripted/Endoscopy/Endoscopy.py

```diff
@@ -1015,7 +1027,7 @@ class EndoscopyPathModel:
     - Add a single polyline
     """
 
-    def __init__(self, resampledCurve, inputCurve, outputPathNode=None):
+    def __init__(self, resampledCurve, inputCurve, cameraNode, outputPathNode=None):
```

Comment:
```suggestion
    def __init__(self, resampledCurve, inputCurve, outputPathNode=None):
```

The `cameraNode` is not used in the function.

Reply:
I will remove it.